example/src/screens/App.tsx

```diff
@@ -1,4 +1,4 @@
-import '../../expo-crypto-shim.js';
+// import '../../expo-crypto-shim.js'; --> Only for Expo 48
```

Comment:
Can we detect version of expo and auto apply this if needed?

Reply:
mmm unless projects have `expo-constants`, i cannot check the sdk version. But maybe i can check if `crypto` exists, and if it doesn't, polyfilll it